01-JS-DOM-APIs/Ejercicios 1,2,3,4/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Javascript basics</title>
+        <meta name="description" content="Javascript basics">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="css/normalize.css">
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <script type="text/javascript" src="js/script.js"></script>
+    </head>
+    <body>
+        <section class="main main_hidden" id="main_hidden">
```

Comment:
The id of the `<section>` sounds weird to me. Try to avoid the relation between an element a particular state of the element.